src/main/groovy/edu/wpi/first/gradlerio/test/JavaTestPlugin.groovy

```diff
@@ -46,9 +46,16 @@ class JavaTestPlugin implements Plugin<Project> {
 
                 def ldpath = jniExtractionDir(project).absolutePath
 
-                if (OperatingSystem.current().isUnix()) {
+                // must check for isMacOsX first, because isUnix() returns true on a mac
+                if (OperatingSystem.current().isMacOsX()) {
+                    if (OperatingSystem.current().toString().contains("10.15")) {
```

Comment:
We need to figure out how to make this future proof, because this would fail on Big Sur.